packages/migrate/src/helpers/strings.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Checks if there are changes in the code between to strings
+ * by comparing code without empty spaces between structures.
+ */
+export function isSourceCodeChanged(originalText: string, updateText: string): boolean {
```

Comment:
Refactoring. This function just compare strings... so moved it out if AST related helpers